servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -482,7 +486,7 @@ def get_last_page(total_results, page_size):
 def page_from_offset(offset, page_size):
     if offset == 0:
         return 1
-    return 1 + (offset / page_size)
+    return math.floor(1 + (offset / page_size))
```

Comment:
Floor is used here to revert to normal integer division, which I think was being relied on before. The tests assume an integer result.